2025-03-11 11:23:51.200 -03:00 [INF] User profile is available. Using 'C:\Users\Renato\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2025-03-11 11:23:52.009 -03:00 [INF] Now listening on: https://localhost:7127
2025-03-11 11:23:52.032 -03:00 [INF] Now listening on: http://localhost:5120
2025-03-11 11:23:52.372 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2025-03-11 11:23:52.380 -03:00 [INF] Hosting environment: Development
2025-03-11 11:23:52.382 -03:00 [INF] Content root path: D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API
2025-03-11 11:23:55.083 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/weatherforecast - null null
2025-03-11 11:23:58.295 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 11:23:58.347 -03:00 [INF] Route matched with {action = "Get", controller = "WeatherForecast"}. Executing controller action with signature System.Collections.Generic.IEnumerable`1[WEB_API.WeatherForecast] Get() on controller WEB_API.Controllers.WeatherForecastController (WEB-API).
2025-03-11 11:23:58.372 -03:00 [INF] Executing ObjectResult, writing value of type 'WEB_API.WeatherForecast[]'.
2025-03-11 11:23:58.425 -03:00 [INF] Executed action WEB_API.Controllers.WeatherForecastController.Get (WEB-API) in 68.5513ms
2025-03-11 11:23:58.428 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 11:23:58.436 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/weatherforecast - 200 null application/json; charset=utf-8 3368.2811ms
2025-03-11 11:23:58.606 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/favicon.ico - null null
2025-03-11 11:23:58.613 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/favicon.ico - 404 0 null 7.8392ms
2025-03-11 11:23:58.621 -03:00 [INF] Request reached the end of the middleware pipeline without being handled by application code. Request path: GET https://localhost:7127/favicon.ico, Response status code: 404
2025-03-11 11:24:16.680 -03:00 [INF] Request starting HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - application/json 56
2025-03-11 11:24:16.838 -03:00 [INF] Failed to validate the token.
Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
 ---> Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken.ReadToken(String encodedJson)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken..ctor(String jwtEncodedString)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebTokenHandler.ReadToken(String token, TokenValidationParameters validationParameters)
   --- End of inner exception stack trace ---
2025-03-11 11:24:16.892 -03:00 [INF] Bearer was not authenticated. Failure message: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
2025-03-11 11:24:16.894 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:24:16.908 -03:00 [INF] Route matched with {action = "Login", controller = "Usuarios"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[System.String]] Login(LoginRequest) on controller WEB_API.Controllers.UsuariosController (WEB-API).
2025-03-11 11:24:23.958 -03:00 [INF] Executed DbCommand (273ms) [Parameters=[@__request_Email_0='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [u].[ID], [u].[Email], [u].[Senha]
FROM [Usuarios] AS [u]
WHERE [u].[Email] = @__request_Email_0
2025-03-11 11:24:24.397 -03:00 [INF] Executing UnauthorizedObjectResult, writing value of type 'System.String'.
2025-03-11 11:24:24.405 -03:00 [INF] Executed action WEB_API.Controllers.UsuariosController.Login (WEB-API) in 7493.7487ms
2025-03-11 11:24:24.408 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:24:24.416 -03:00 [INF] Request finished HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - 401 null text/plain; charset=utf-8 7736.4222ms
2025-03-11 11:27:25.966 -03:00 [INF] Request starting HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - application/json 56
2025-03-11 11:27:25.972 -03:00 [INF] Failed to validate the token.
Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
 ---> Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken.ReadToken(String encodedJson)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken..ctor(String jwtEncodedString)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebTokenHandler.ReadToken(String token, TokenValidationParameters validationParameters)
   --- End of inner exception stack trace ---
2025-03-11 11:27:25.979 -03:00 [INF] Bearer was not authenticated. Failure message: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
2025-03-11 11:27:25.981 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:27:25.982 -03:00 [INF] Route matched with {action = "Login", controller = "Usuarios"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[System.String]] Login(LoginRequest) on controller WEB_API.Controllers.UsuariosController (WEB-API).
2025-03-11 11:27:26.149 -03:00 [INF] Executed DbCommand (28ms) [Parameters=[@__request_Email_0='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [u].[ID], [u].[Email], [u].[Senha]
FROM [Usuarios] AS [u]
WHERE [u].[Email] = @__request_Email_0
2025-03-11 11:27:26.296 -03:00 [INF] Executing UnauthorizedObjectResult, writing value of type 'System.String'.
2025-03-11 11:27:26.300 -03:00 [INF] Executed action WEB_API.Controllers.UsuariosController.Login (WEB-API) in 312.6078ms
2025-03-11 11:27:26.303 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:27:26.304 -03:00 [INF] Request finished HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - 401 null text/plain; charset=utf-8 337.6855ms
2025-03-11 11:28:18.905 -03:00 [INF] User profile is available. Using 'C:\Users\Renato\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2025-03-11 11:28:19.167 -03:00 [INF] Now listening on: https://localhost:7127
2025-03-11 11:28:19.169 -03:00 [INF] Now listening on: http://localhost:5120
2025-03-11 11:28:19.264 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2025-03-11 11:28:19.266 -03:00 [INF] Hosting environment: Development
2025-03-11 11:28:19.268 -03:00 [INF] Content root path: D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API
2025-03-11 11:28:20.268 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/weatherforecast - null null
2025-03-11 11:28:21.942 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 11:28:21.960 -03:00 [INF] Route matched with {action = "Get", controller = "WeatherForecast"}. Executing controller action with signature System.Collections.Generic.IEnumerable`1[WEB_API.WeatherForecast] Get() on controller WEB_API.Controllers.WeatherForecastController (WEB-API).
2025-03-11 11:28:21.970 -03:00 [INF] Executing ObjectResult, writing value of type 'WEB_API.WeatherForecast[]'.
2025-03-11 11:28:21.992 -03:00 [INF] Executed action WEB_API.Controllers.WeatherForecastController.Get (WEB-API) in 26.1115ms
2025-03-11 11:28:21.996 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 11:28:22.001 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/weatherforecast - 200 null application/json; charset=utf-8 1747.7967ms
2025-03-11 11:28:38.906 -03:00 [INF] Request starting HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - application/json 56
2025-03-11 11:28:38.930 -03:00 [INF] Failed to validate the token.
Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
 ---> Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken.ReadToken(String encodedJson)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken..ctor(String jwtEncodedString)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebTokenHandler.ReadToken(String token, TokenValidationParameters validationParameters)
   --- End of inner exception stack trace ---
2025-03-11 11:28:38.968 -03:00 [INF] Bearer was not authenticated. Failure message: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
2025-03-11 11:28:38.971 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:28:38.985 -03:00 [INF] Route matched with {action = "Login", controller = "Usuarios"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[System.String]] Login(LoginRequest) on controller WEB_API.Controllers.UsuariosController (WEB-API).
2025-03-11 11:28:40.317 -03:00 [INF] Executed DbCommand (66ms) [Parameters=[@__request_Email_0='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [u].[ID], [u].[Email], [u].[Senha]
FROM [Usuarios] AS [u]
WHERE [u].[Email] = @__request_Email_0
2025-03-11 11:28:40.832 -03:00 [INF] Executing UnauthorizedObjectResult, writing value of type 'System.String'.
2025-03-11 11:28:40.837 -03:00 [INF] Executed action WEB_API.Controllers.UsuariosController.Login (WEB-API) in 1847.1206ms
2025-03-11 11:28:40.839 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:28:40.849 -03:00 [INF] Request finished HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - 401 null text/plain; charset=utf-8 1943.156ms
2025-03-11 11:28:53.975 -03:00 [INF] Request starting HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - application/json 57
2025-03-11 11:28:53.981 -03:00 [INF] Failed to validate the token.
Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
 ---> Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken.ReadToken(String encodedJson)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken..ctor(String jwtEncodedString)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebTokenHandler.ReadToken(String token, TokenValidationParameters validationParameters)
   --- End of inner exception stack trace ---
2025-03-11 11:28:53.986 -03:00 [INF] Bearer was not authenticated. Failure message: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
2025-03-11 11:28:53.987 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:28:53.989 -03:00 [INF] Route matched with {action = "Login", controller = "Usuarios"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[System.String]] Login(LoginRequest) on controller WEB_API.Controllers.UsuariosController (WEB-API).
2025-03-11 11:28:54.016 -03:00 [INF] Executed DbCommand (3ms) [Parameters=[@__request_Email_0='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [u].[ID], [u].[Email], [u].[Senha]
FROM [Usuarios] AS [u]
WHERE [u].[Email] = @__request_Email_0
2025-03-11 11:28:54.454 -03:00 [INF] Executing OkObjectResult, writing value of type 'System.String'.
2025-03-11 11:28:54.457 -03:00 [INF] Executed action WEB_API.Controllers.UsuariosController.Login (WEB-API) in 466.1191ms
2025-03-11 11:28:54.459 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:28:54.463 -03:00 [INF] Request finished HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - 200 null text/plain; charset=utf-8 487.5261ms
2025-03-11 11:40:27.597 -03:00 [INF] User profile is available. Using 'C:\Users\Renato\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2025-03-11 11:40:27.947 -03:00 [INF] Now listening on: https://localhost:7127
2025-03-11 11:40:27.950 -03:00 [INF] Now listening on: http://localhost:5120
2025-03-11 11:40:28.121 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2025-03-11 11:40:28.133 -03:00 [INF] Hosting environment: Development
2025-03-11 11:40:28.135 -03:00 [INF] Content root path: D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API
2025-03-11 11:40:28.462 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/weatherforecast - null null
2025-03-11 11:40:28.663 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 11:40:28.682 -03:00 [INF] Route matched with {action = "Get", controller = "WeatherForecast"}. Executing controller action with signature System.Collections.Generic.IEnumerable`1[WEB_API.WeatherForecast] Get() on controller WEB_API.Controllers.WeatherForecastController (WEB-API).
2025-03-11 11:40:28.695 -03:00 [INF] Executing ObjectResult, writing value of type 'WEB_API.WeatherForecast[]'.
2025-03-11 11:40:28.717 -03:00 [INF] Executed action WEB_API.Controllers.WeatherForecastController.Get (WEB-API) in 28.526ms
2025-03-11 11:40:28.722 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 11:40:28.728 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/weatherforecast - 200 null application/json; charset=utf-8 279.3843ms
2025-03-11 11:40:32.160 -03:00 [INF] Request starting HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - application/json 57
2025-03-11 11:40:32.184 -03:00 [INF] Failed to validate the token.
Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
 ---> Microsoft.IdentityModel.Tokens.SecurityTokenMalformedException: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken.ReadToken(String encodedJson)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebToken..ctor(String jwtEncodedString)
   at Microsoft.IdentityModel.JsonWebTokens.JsonWebTokenHandler.ReadToken(String token, TokenValidationParameters validationParameters)
   --- End of inner exception stack trace ---
2025-03-11 11:40:32.235 -03:00 [INF] Bearer was not authenticated. Failure message: IDX14100: JWT is not well formed, there are no dots (.).
The token needs to be in JWS or JWE Compact Serialization Format. (JWS): 'EncodedHeader.EndcodedPayload.EncodedSignature'. (JWE): 'EncodedProtectedHeader.EncodedEncryptedKey.EncodedInitializationVector.EncodedCiphertext.EncodedAuthenticationTag'.
2025-03-11 11:40:32.236 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:40:32.248 -03:00 [INF] Route matched with {action = "Login", controller = "Usuarios"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult] Login(LoginRequest) on controller WEB_API.Controllers.UsuariosController (WEB-API).
2025-03-11 11:40:33.849 -03:00 [INF] Executed DbCommand (87ms) [Parameters=[@__request_Email_0='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [u].[ID], [u].[Email], [u].[Senha]
FROM [Usuarios] AS [u]
WHERE [u].[Email] = @__request_Email_0
2025-03-11 11:40:34.398 -03:00 [INF] Executing OkObjectResult, writing value of type '<>f__AnonymousType2`3[[System.String, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.Boolean, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.String, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]'.
2025-03-11 11:40:34.407 -03:00 [INF] Executed action WEB_API.Controllers.UsuariosController.Login (WEB-API) in 2155.8553ms
2025-03-11 11:40:34.410 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.UsuariosController.Login (WEB-API)'
2025-03-11 11:40:34.421 -03:00 [INF] Request finished HTTP/1.1 POST https://localhost:7127/api/Usuarios/login - 200 null application/json; charset=utf-8 2260.8567ms
2025-03-11 11:40:56.281 -03:00 [INF] Request starting HTTP/1.1 GET https://localhost:7127/api/produtos - null null
2025-03-11 11:40:56.452 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.ProdutosController.GetProdutos (WEB-API)'
2025-03-11 11:40:56.458 -03:00 [INF] Route matched with {action = "GetProdutos", controller = "Produtos"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[WEB_API.Models.Produto]]] GetProdutos() on controller WEB_API.Controllers.ProdutosController (WEB-API).
2025-03-11 11:40:56.503 -03:00 [INF] Executed DbCommand (10ms) [Parameters=[], CommandType='"Text"', CommandTimeout='30']
SELECT [p].[Id], [p].[Imagem], [p].[Nome], [p].[Preco]
FROM [Produtos] AS [p]
2025-03-11 11:40:56.525 -03:00 [INF] Executing ObjectResult, writing value of type 'System.Collections.Generic.List`1[[WEB_API.Models.Produto, WEB-API, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]'.
2025-03-11 11:40:56.548 -03:00 [INF] Executed action WEB_API.Controllers.ProdutosController.GetProdutos (WEB-API) in 87.6005ms
2025-03-11 11:40:56.552 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.ProdutosController.GetProdutos (WEB-API)'
2025-03-11 11:40:56.555 -03:00 [INF] Request finished HTTP/1.1 GET https://localhost:7127/api/produtos - 200 null application/json; charset=utf-8 273.7348ms
2025-03-11 11:41:07.151 -03:00 [INF] Request starting HTTP/1.1 GET https://localhost:7127/api/produtos/2 - null null
2025-03-11 11:41:07.158 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 11:41:07.165 -03:00 [INF] Route matched with {action = "GetProduto", controller = "Produtos"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[WEB_API.Models.Produto]] GetProduto(Int32) on controller WEB_API.Controllers.ProdutosController (WEB-API).
2025-03-11 11:41:07.443 -03:00 [INF] Executed DbCommand (7ms) [Parameters=[@__p_0='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [p].[Id], [p].[Imagem], [p].[Nome], [p].[Preco]
FROM [Produtos] AS [p]
WHERE [p].[Id] = @__p_0
2025-03-11 11:41:07.448 -03:00 [INF] Executing ObjectResult, writing value of type 'Microsoft.AspNetCore.Mvc.ProblemDetails'.
2025-03-11 11:41:07.519 -03:00 [INF] Executed action WEB_API.Controllers.ProdutosController.GetProduto (WEB-API) in 350.9048ms
2025-03-11 11:41:07.523 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 11:41:07.524 -03:00 [INF] Request finished HTTP/1.1 GET https://localhost:7127/api/produtos/2 - 404 null application/problem+json; charset=utf-8 373.0779ms
2025-03-11 11:55:29.777 -03:00 [INF] User profile is available. Using 'C:\Users\Renato\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2025-03-11 11:55:30.256 -03:00 [INF] Now listening on: https://localhost:7127
2025-03-11 11:55:30.258 -03:00 [INF] Now listening on: http://localhost:5120
2025-03-11 11:55:30.417 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2025-03-11 11:55:30.418 -03:00 [INF] Hosting environment: Development
2025-03-11 11:55:30.419 -03:00 [INF] Content root path: D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API
2025-03-11 11:55:31.045 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/weatherforecast - null null
2025-03-11 11:55:31.250 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 11:55:31.271 -03:00 [INF] Route matched with {action = "Get", controller = "WeatherForecast"}. Executing controller action with signature System.Collections.Generic.IEnumerable`1[WEB_API.WeatherForecast] Get() on controller WEB_API.Controllers.WeatherForecastController (WEB-API).
2025-03-11 11:55:31.285 -03:00 [INF] Executing ObjectResult, writing value of type 'WEB_API.WeatherForecast[]'.
2025-03-11 11:55:31.312 -03:00 [INF] Executed action WEB_API.Controllers.WeatherForecastController.Get (WEB-API) in 34.3461ms
2025-03-11 11:55:31.316 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 11:55:31.321 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/weatherforecast - 200 null application/json; charset=utf-8 284.7451ms
2025-03-11 11:55:35.858 -03:00 [INF] Request starting HTTP/1.1 GET https://localhost:7127/api/produtos/2 - null null
2025-03-11 11:55:35.944 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 11:55:35.959 -03:00 [INF] Route matched with {action = "GetProduto", controller = "Produtos"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[WEB_API.Models.Produto]] GetProduto(Int32) on controller WEB_API.Controllers.ProdutosController (WEB-API).
2025-03-11 11:55:37.301 -03:00 [INF] Executed DbCommand (53ms) [Parameters=[@__p_0='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [p].[Id], [p].[Imagem], [p].[Nome], [p].[Preco]
FROM [Produtos] AS [p]
WHERE [p].[Id] = @__p_0
2025-03-11 11:55:37.330 -03:00 [ERR] Erro ao obter o produto com ID 2.
System.Exception: Produto não encontrado
2025-03-11 11:55:37.726 -03:00 [ERR] Failed executing DbCommand (70ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [LogsErros] ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);
2025-03-11 11:55:37.938 -03:00 [ERR] An exception occurred in the database while saving changes for context type 'WEB_API.Models.ProdutosUsuariosContext'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
2025-03-11 11:55:38.159 -03:00 [ERR] Erro ao obter o produto com ID 2.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at WEB_API.Services.LogErroService.LogErroAsync(Exception ex, String enderecoRequisicao, String metodoRequisicao, String corpoRequisicao, String nomeControlador) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Services\LogErroService.cs:line 25
   at WEB_API.Controllers.ProdutosController.GetProduto(Int32 id) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Controllers\ProdutosController.cs:line 49
2025-03-11 11:55:38.210 -03:00 [ERR] Failed executing DbCommand (8ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000), @p7='?' (Size = 4000), @p8='?' (DbType = DateTime2), @p9='?' (Size = 4000), @p10='?' (Size = 4000), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
MERGE [LogsErros] USING (
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, 0),
(@p7, @p8, @p9, @p10, @p11, @p12, @p13, 1)) AS i ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace], _Position) ON 1=0
WHEN NOT MATCHED THEN
INSERT ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
VALUES (i.[CorpoRequisicao], i.[DataHora], i.[EnderecoRequisicao], i.[Mensagem], i.[MetodoRequisicao], i.[NomeControlador], i.[StackTrace])
OUTPUT INSERTED.[Id], i._Position;
2025-03-11 11:55:38.216 -03:00 [ERR] An exception occurred in the database while saving changes for context type 'WEB_API.Models.ProdutosUsuariosContext'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
2025-03-11 11:55:38.321 -03:00 [INF] Executed action WEB_API.Controllers.ProdutosController.GetProduto (WEB-API) in 2359.3464ms
2025-03-11 11:55:38.326 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 11:55:38.436 -03:00 [ERR] Erro não tratado: An error occurred while saving the entity changes. See the inner exception for details.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at WEB_API.Services.LogErroService.LogErroAsync(Exception ex, String enderecoRequisicao, String metodoRequisicao, String corpoRequisicao, String nomeControlador) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Services\LogErroService.cs:line 25
   at WEB_API.Controllers.ProdutosController.GetProduto(Int32 id) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Controllers\ProdutosController.cs:line 58
   at lambda_method7(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|7_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WEB_API.Middlewares.ErrorHandlerMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\ErrorHandlerMiddleware.cs:line 17
2025-03-11 11:55:38.481 -03:00 [ERR] Failed executing DbCommand (3ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = -1)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [LogsErros] ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);
2025-03-11 11:55:38.487 -03:00 [ERR] An exception occurred in the database while saving changes for context type 'WEB_API.Models.ProdutosUsuariosContext'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
2025-03-11 11:55:38.807 -03:00 [ERR] An unhandled exception has occurred while executing the request.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at WEB_API.Services.LogErroService.LogErroAsync(Exception ex, String enderecoRequisicao, String metodoRequisicao, String corpoRequisicao, String nomeControlador) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Services\LogErroService.cs:line 25
   at WEB_API.Middlewares.ErrorHandlerMiddleware.HandleExceptionAsync(HttpContext context, Exception ex) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\ErrorHandlerMiddleware.cs:line 53
   at WEB_API.Middlewares.ErrorHandlerMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\ErrorHandlerMiddleware.cs:line 21
   at WEB_API.Middlewares.JwtMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\JwtMiddleware.cs:line 15
   at WEB_API.Middlewares.LogServiceMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\LogServiceMiddleware.cs:line 18
   at WEB_API.Middlewares.DbContextMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\DbContextMiddleware.cs:line 17
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2025-03-11 11:55:38.834 -03:00 [INF] Request finished HTTP/1.1 GET https://localhost:7127/api/produtos/2 - 500 null text/plain; charset=utf-8 2975.3775ms
2025-03-11 11:56:02.038 -03:00 [INF] Request starting HTTP/1.1 GET https://localhost:7127/api/produtos/2 - null null
2025-03-11 11:56:02.048 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 11:56:02.051 -03:00 [INF] Route matched with {action = "GetProduto", controller = "Produtos"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[WEB_API.Models.Produto]] GetProduto(Int32) on controller WEB_API.Controllers.ProdutosController (WEB-API).
2025-03-11 11:56:11.620 -03:00 [INF] Executed DbCommand (3ms) [Parameters=[@__p_0='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [p].[Id], [p].[Imagem], [p].[Nome], [p].[Preco]
FROM [Produtos] AS [p]
WHERE [p].[Id] = @__p_0
2025-03-11 11:56:18.077 -03:00 [ERR] Erro ao obter o produto com ID 2.
System.Exception: Produto não encontrado
2025-03-11 11:56:20.261 -03:00 [ERR] Failed executing DbCommand (21ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [LogsErros] ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);
2025-03-11 11:56:20.269 -03:00 [ERR] An exception occurred in the database while saving changes for context type 'WEB_API.Models.ProdutosUsuariosContext'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
2025-03-11 11:56:26.960 -03:00 [ERR] Erro ao obter o produto com ID 2.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at WEB_API.Services.LogErroService.LogErroAsync(Exception ex, String enderecoRequisicao, String metodoRequisicao, String corpoRequisicao, String nomeControlador) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Services\LogErroService.cs:line 25
   at WEB_API.Controllers.ProdutosController.GetProduto(Int32 id) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Controllers\ProdutosController.cs:line 49
2025-03-11 11:56:27.608 -03:00 [ERR] Failed executing DbCommand (40ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000), @p7='?' (Size = 4000), @p8='?' (DbType = DateTime2), @p9='?' (Size = 4000), @p10='?' (Size = 4000), @p11='?' (Size = 4000), @p12='?' (Size = 4000), @p13='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
MERGE [LogsErros] USING (
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6, 0),
(@p7, @p8, @p9, @p10, @p11, @p12, @p13, 1)) AS i ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace], _Position) ON 1=0
WHEN NOT MATCHED THEN
INSERT ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
VALUES (i.[CorpoRequisicao], i.[DataHora], i.[EnderecoRequisicao], i.[Mensagem], i.[MetodoRequisicao], i.[NomeControlador], i.[StackTrace])
OUTPUT INSERTED.[Id], i._Position;
2025-03-11 11:56:27.616 -03:00 [ERR] An exception occurred in the database while saving changes for context type 'WEB_API.Models.ProdutosUsuariosContext'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
2025-03-11 11:56:27.827 -03:00 [INF] Executed action WEB_API.Controllers.ProdutosController.GetProduto (WEB-API) in 25769.9291ms
2025-03-11 11:56:27.829 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 11:56:27.926 -03:00 [ERR] Erro não tratado: An error occurred while saving the entity changes. See the inner exception for details.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at WEB_API.Services.LogErroService.LogErroAsync(Exception ex, String enderecoRequisicao, String metodoRequisicao, String corpoRequisicao, String nomeControlador) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Services\LogErroService.cs:line 25
   at WEB_API.Controllers.ProdutosController.GetProduto(Int32 id) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Controllers\ProdutosController.cs:line 58
   at lambda_method7(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|7_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at WEB_API.Middlewares.ErrorHandlerMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\ErrorHandlerMiddleware.cs:line 17
2025-03-11 11:56:28.020 -03:00 [ERR] Failed executing DbCommand (62ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = -1)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [LogsErros] ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);
2025-03-11 11:56:28.070 -03:00 [ERR] An exception occurred in the database while saving changes for context type 'WEB_API.Models.ProdutosUsuariosContext'.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
2025-03-11 11:56:28.500 -03:00 [ERR] An unhandled exception has occurred while executing the request.
Microsoft.EntityFrameworkCore.DbUpdateException: An error occurred while saving the entity changes. See the inner exception for details.
 ---> Microsoft.Data.SqlClient.SqlException (0x80131904): Nome de coluna 'NomeControlador' inválido.
   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__211_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
ClientConnectionId:e9762485-c51a-46d9-832f-cf5f0f7daad6
Error Number:207,State:1,Class:16
   --- End of inner exception stack trace ---
   at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Update.Internal.SqlServerModificationCommandBatch.ExecuteAsync(IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChangesAsync(IList`1 entries, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at WEB_API.Services.LogErroService.LogErroAsync(Exception ex, String enderecoRequisicao, String metodoRequisicao, String corpoRequisicao, String nomeControlador) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Services\LogErroService.cs:line 25
   at WEB_API.Middlewares.ErrorHandlerMiddleware.HandleExceptionAsync(HttpContext context, Exception ex) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\ErrorHandlerMiddleware.cs:line 53
   at WEB_API.Middlewares.ErrorHandlerMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\ErrorHandlerMiddleware.cs:line 21
   at WEB_API.Middlewares.JwtMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\JwtMiddleware.cs:line 15
   at WEB_API.Middlewares.LogServiceMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\LogServiceMiddleware.cs:line 18
   at WEB_API.Middlewares.DbContextMiddleware.InvokeAsync(HttpContext context) in D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API\Middlewares\DbContextMiddleware.cs:line 17
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2025-03-11 11:56:28.593 -03:00 [INF] Request finished HTTP/1.1 GET https://localhost:7127/api/produtos/2 - 500 null text/plain; charset=utf-8 26555.2319ms
2025-03-11 12:05:26.943 -03:00 [INF] User profile is available. Using 'C:\Users\Renato\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2025-03-11 12:05:27.223 -03:00 [INF] Now listening on: https://localhost:7127
2025-03-11 12:05:27.225 -03:00 [INF] Now listening on: http://localhost:5120
2025-03-11 12:05:27.384 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2025-03-11 12:05:27.385 -03:00 [INF] Hosting environment: Development
2025-03-11 12:05:27.386 -03:00 [INF] Content root path: D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API
2025-03-11 12:05:27.736 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/weatherforecast - null null
2025-03-11 12:05:27.947 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 12:05:27.969 -03:00 [INF] Route matched with {action = "Get", controller = "WeatherForecast"}. Executing controller action with signature System.Collections.Generic.IEnumerable`1[WEB_API.WeatherForecast] Get() on controller WEB_API.Controllers.WeatherForecastController (WEB-API).
2025-03-11 12:05:27.979 -03:00 [INF] Executing ObjectResult, writing value of type 'WEB_API.WeatherForecast[]'.
2025-03-11 12:05:28.009 -03:00 [INF] Executed action WEB_API.Controllers.WeatherForecastController.Get (WEB-API) in 33.5821ms
2025-03-11 12:05:28.013 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 12:05:28.019 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/weatherforecast - 200 null application/json; charset=utf-8 294.0601ms
2025-03-11 12:05:31.541 -03:00 [INF] Request starting HTTP/1.1 GET https://localhost:7127/api/produtos/2 - null null
2025-03-11 12:05:31.623 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 12:05:31.636 -03:00 [INF] Route matched with {action = "GetProduto", controller = "Produtos"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[WEB_API.Models.Produto]] GetProduto(Int32) on controller WEB_API.Controllers.ProdutosController (WEB-API).
2025-03-11 12:05:33.231 -03:00 [INF] Executed DbCommand (53ms) [Parameters=[@__p_0='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [p].[Id], [p].[Imagem], [p].[Nome], [p].[Preco]
FROM [Produtos] AS [p]
WHERE [p].[Id] = @__p_0
2025-03-11 12:05:33.254 -03:00 [ERR] Erro ao obter o produto com ID 2.
System.Exception: Produto não encontrado
2025-03-11 12:05:33.413 -03:00 [INF] Executed DbCommand (13ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [LogsErros] ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);
2025-03-11 12:05:33.552 -03:00 [INF] Executing NotFoundObjectResult, writing value of type '<>f__AnonymousType0`2[[System.Int32, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.String, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]'.
2025-03-11 12:05:33.563 -03:00 [INF] Executed action WEB_API.Controllers.ProdutosController.GetProduto (WEB-API) in 1922.6735ms
2025-03-11 12:05:33.567 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 12:05:33.575 -03:00 [INF] Request finished HTTP/1.1 GET https://localhost:7127/api/produtos/2 - 404 null application/json; charset=utf-8 2034.2987ms
2025-03-11 12:12:18.666 -03:00 [INF] User profile is available. Using 'C:\Users\Renato\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2025-03-11 12:12:18.976 -03:00 [INF] Now listening on: https://localhost:7127
2025-03-11 12:12:18.979 -03:00 [INF] Now listening on: http://localhost:5120
2025-03-11 12:12:19.070 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2025-03-11 12:12:19.072 -03:00 [INF] Hosting environment: Development
2025-03-11 12:12:19.074 -03:00 [INF] Content root path: D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API
2025-03-11 12:12:19.700 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/weatherforecast - null null
2025-03-11 12:12:19.914 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 12:12:19.936 -03:00 [INF] Route matched with {action = "Get", controller = "WeatherForecast"}. Executing controller action with signature System.Collections.Generic.IEnumerable`1[WEB_API.WeatherForecast] Get() on controller WEB_API.Controllers.WeatherForecastController (WEB-API).
2025-03-11 12:12:19.945 -03:00 [INF] Executing ObjectResult, writing value of type 'WEB_API.WeatherForecast[]'.
2025-03-11 12:12:19.969 -03:00 [INF] Executed action WEB_API.Controllers.WeatherForecastController.Get (WEB-API) in 29.2194ms
2025-03-11 12:12:19.973 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 12:12:19.979 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/weatherforecast - 200 null application/json; charset=utf-8 290.521ms
2025-03-11 12:12:24.340 -03:00 [INF] Request starting HTTP/1.1 GET https://localhost:7127/api/produtos/2 - null null
2025-03-11 12:12:24.444 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 12:12:24.458 -03:00 [INF] Route matched with {action = "GetProduto", controller = "Produtos"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[WEB_API.Models.Produto]] GetProduto(Int32) on controller WEB_API.Controllers.ProdutosController (WEB-API).
2025-03-11 12:12:25.823 -03:00 [INF] Executed DbCommand (57ms) [Parameters=[@__p_0='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [p].[Id], [p].[Imagem], [p].[Nome], [p].[Preco]
FROM [Produtos] AS [p]
WHERE [p].[Id] = @__p_0
2025-03-11 12:12:25.848 -03:00 [ERR] Erro ao obter o produto com ID 2.
System.Exception: Produto não encontrado
2025-03-11 12:12:25.999 -03:00 [INF] Executed DbCommand (8ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [LogsErros] ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);
2025-03-11 12:12:26.030 -03:00 [INF] Executing NotFoundObjectResult, writing value of type '<>f__AnonymousType0`2[[System.Int32, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.String, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]'.
2025-03-11 12:12:26.039 -03:00 [INF] Executed action WEB_API.Controllers.ProdutosController.GetProduto (WEB-API) in 1578.844ms
2025-03-11 12:12:26.042 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 12:12:26.052 -03:00 [INF] Request finished HTTP/1.1 GET https://localhost:7127/api/produtos/2 - 404 null application/json; charset=utf-8 1712.2682ms
2025-03-11 12:12:34.723 -03:00 [INF] Request starting HTTP/1.1 GET https://localhost:7127/api/produtos/100 - null null
2025-03-11 12:12:34.734 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 12:12:34.735 -03:00 [INF] Route matched with {action = "GetProduto", controller = "Produtos"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[WEB_API.Models.Produto]] GetProduto(Int32) on controller WEB_API.Controllers.ProdutosController (WEB-API).
2025-03-11 12:12:34.756 -03:00 [INF] Executed DbCommand (2ms) [Parameters=[@__p_0='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [p].[Id], [p].[Imagem], [p].[Nome], [p].[Preco]
FROM [Produtos] AS [p]
WHERE [p].[Id] = @__p_0
2025-03-11 12:12:34.758 -03:00 [ERR] Erro ao obter o produto com ID 100.
System.Exception: Produto não encontrado
2025-03-11 12:12:34.766 -03:00 [INF] Executed DbCommand (2ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [LogsErros] ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);
2025-03-11 12:12:34.770 -03:00 [INF] Executing NotFoundObjectResult, writing value of type '<>f__AnonymousType0`2[[System.Int32, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.String, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]'.
2025-03-11 12:12:34.771 -03:00 [INF] Executed action WEB_API.Controllers.ProdutosController.GetProduto (WEB-API) in 34.6284ms
2025-03-11 12:12:34.773 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 12:12:34.774 -03:00 [INF] Request finished HTTP/1.1 GET https://localhost:7127/api/produtos/100 - 404 null application/json; charset=utf-8 50.6221ms
2025-03-11 12:12:40.148 -03:00 [INF] Request starting HTTP/1.1 GET https://localhost:7127/api/produtos/00 - null null
2025-03-11 12:12:40.154 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 12:12:40.156 -03:00 [INF] Route matched with {action = "GetProduto", controller = "Produtos"}. Executing controller action with signature System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.ActionResult`1[WEB_API.Models.Produto]] GetProduto(Int32) on controller WEB_API.Controllers.ProdutosController (WEB-API).
2025-03-11 12:12:40.225 -03:00 [INF] Executed DbCommand (2ms) [Parameters=[@__p_0='?' (DbType = Int32)], CommandType='"Text"', CommandTimeout='30']
SELECT TOP(1) [p].[Id], [p].[Imagem], [p].[Nome], [p].[Preco]
FROM [Produtos] AS [p]
WHERE [p].[Id] = @__p_0
2025-03-11 12:12:40.228 -03:00 [ERR] Erro ao obter o produto com ID 0.
System.Exception: Produto não encontrado
2025-03-11 12:12:40.230 -03:00 [INF] Executed DbCommand (1ms) [Parameters=[@p0='?' (Size = 4000), @p1='?' (DbType = DateTime2), @p2='?' (Size = 4000), @p3='?' (Size = 4000), @p4='?' (Size = 4000), @p5='?' (Size = 4000), @p6='?' (Size = 4000)], CommandType='"Text"', CommandTimeout='30']
SET IMPLICIT_TRANSACTIONS OFF;
SET NOCOUNT ON;
INSERT INTO [LogsErros] ([CorpoRequisicao], [DataHora], [EnderecoRequisicao], [Mensagem], [MetodoRequisicao], [NomeControlador], [StackTrace])
OUTPUT INSERTED.[Id]
VALUES (@p0, @p1, @p2, @p3, @p4, @p5, @p6);
2025-03-11 12:12:40.234 -03:00 [INF] Executing NotFoundObjectResult, writing value of type '<>f__AnonymousType0`2[[System.Int32, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e],[System.String, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]'.
2025-03-11 12:12:40.236 -03:00 [INF] Executed action WEB_API.Controllers.ProdutosController.GetProduto (WEB-API) in 77.2318ms
2025-03-11 12:12:40.238 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.ProdutosController.GetProduto (WEB-API)'
2025-03-11 12:12:40.240 -03:00 [INF] Request finished HTTP/1.1 GET https://localhost:7127/api/produtos/00 - 404 null application/json; charset=utf-8 91.6728ms
2025-03-11 17:09:14.500 -03:00 [INF] User profile is available. Using 'C:\Users\Renato\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2025-03-11 17:09:14.957 -03:00 [INF] Now listening on: https://localhost:7127
2025-03-11 17:09:14.960 -03:00 [INF] Now listening on: http://localhost:5120
2025-03-11 17:09:15.183 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2025-03-11 17:09:15.184 -03:00 [INF] Hosting environment: Development
2025-03-11 17:09:15.185 -03:00 [INF] Content root path: D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API
2025-03-11 17:09:16.781 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/weatherforecast - null null
2025-03-11 17:09:17.182 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 17:09:17.222 -03:00 [INF] Route matched with {action = "Get", controller = "WeatherForecast"}. Executing controller action with signature System.Collections.Generic.IEnumerable`1[WEB_API.WeatherForecast] Get() on controller WEB_API.Controllers.WeatherForecastController (WEB-API).
2025-03-11 17:09:17.243 -03:00 [INF] Executing ObjectResult, writing value of type 'WEB_API.WeatherForecast[]'.
2025-03-11 17:09:17.283 -03:00 [INF] Executed action WEB_API.Controllers.WeatherForecastController.Get (WEB-API) in 52.3431ms
2025-03-11 17:09:17.292 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 17:09:17.299 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/weatherforecast - 200 null application/json; charset=utf-8 532.3655ms
2025-03-11 18:25:39.102 -03:00 [INF] User profile is available. Using 'C:\Users\Renato\AppData\Local\ASP.NET\DataProtection-Keys' as key repository and Windows DPAPI to encrypt keys at rest.
2025-03-11 18:25:39.383 -03:00 [INF] Now listening on: https://localhost:7127
2025-03-11 18:25:39.386 -03:00 [INF] Now listening on: http://localhost:5120
2025-03-11 18:25:39.518 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2025-03-11 18:25:39.526 -03:00 [INF] Hosting environment: Development
2025-03-11 18:25:39.529 -03:00 [INF] Content root path: D:\Projetos\APS.NET\WEB API\WEB-API\WEB-API
2025-03-11 18:25:40.173 -03:00 [INF] Request starting HTTP/2 GET https://localhost:7127/weatherforecast - null null
2025-03-11 18:25:41.717 -03:00 [INF] Executing endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 18:25:41.739 -03:00 [INF] Route matched with {action = "Get", controller = "WeatherForecast"}. Executing controller action with signature System.Collections.Generic.IEnumerable`1[WEB_API.WeatherForecast] Get() on controller WEB_API.Controllers.WeatherForecastController (WEB-API).
2025-03-11 18:25:41.754 -03:00 [INF] Executing ObjectResult, writing value of type 'WEB_API.WeatherForecast[]'.
2025-03-11 18:25:41.795 -03:00 [INF] Executed action WEB_API.Controllers.WeatherForecastController.Get (WEB-API) in 46.7769ms
2025-03-11 18:25:41.804 -03:00 [INF] Executed endpoint 'WEB_API.Controllers.WeatherForecastController.Get (WEB-API)'
2025-03-11 18:25:41.813 -03:00 [INF] Request finished HTTP/2 GET https://localhost:7127/weatherforecast - 200 null application/json; charset=utf-8 1650.4986ms
